sunxi-boot0.c

```diff
@@ -199,6 +206,39 @@ static int8_t register_mappings[NR_SOC_TYPES][NR_DRAM_PARAMS] = {
 	}
 };
 
+void output_boot0_dram_para(const uint32_t *para, FILE *stream, char const* indent)
+{
+	uint32_t i;
+
+	for (i = 0; i < NR_SOC_TYPES; i++)
+		fprintf(stream, "%12s", soc_name[i]);
+
+	fprintf(stream, "\n");
+
+	for (i = 0; i < NR_DRAM_PARAMS; i++) {
+		int ver;
+
+		for (ver = 0; ver < NR_SOC_TYPES; ver++)
+			if (register_mappings[ver][i] &&
+			    para[register_mappings[ver][i] - 1])
+				break;
+		/* if this parameter is unused, skip the output */
+		if (ver == NR_SOC_TYPES)
+			continue;
+
+		fprintf(stream, "%s\t\t%-12s:", indent, param_name[i]);
```

Comment:
I think here it's one tab too many. Either you ditch the idea of "indent" here, or just have one tab afterwards.